### PR TITLE
 remove pv1.18 add to unit test

### PR DESCRIPTION
**Pull request by jgrant22**

### CODE REVIEW

File: src/test/java/io/github/linuxforhealth/hl7/segments/Hl7EncounterFHIRConversionTest.java [hunk 1]
```diff
@@ -75,6 +75,10 @@ public void test_encounter_visitdescription_present() {
         assertThat(encounterResource).hasSize(1);
 
         Encounter encounter = ResourceUtils.getResourceEncounter(encounterResource.get(0), context);
+        assertThat(encounter.getTypeFirstRep().getCodingFirstRep().getCode()).isEqualTo("A");
```
Comment (LisaWellman) [inline src/test/java/io/github/linuxforhealth/hl7/segments/Hl7EncounterFHIRConversionTest.java:78]: Add a comment these are for PV1-4

File: src/test/java/io/github/linuxforhealth/hl7/segments/Hl7EncounterFHIRConversionTest.java [hunk 2]
```diff
@@ -113,6 +117,7 @@ public void test_encounter_visitdescription_missing() {
         assertThat(encounterResource).hasSize(1);
 
         Encounter encounter = ResourceUtils.getResourceEncounter(encounterResource.get(0), context);
+        assertThat(encounter.getTypeFirstRep()).isNull();
```
Comment (LisaWellman) [inline src/test/java/io/github/linuxforhealth/hl7/segments/Hl7EncounterFHIRConversionTest.java:120]: add a comment that this is to verify PV1-18 is not picked up

**Review by LisaWellman — APPROVED** [review]
Requesting a couple of code comments, but looks good!

File: src/test/java/io/github/linuxforhealth/hl7/segments/Hl7EncounterFHIRConversionTest.java (hunk 1 above)
Comment (jgrant22) [reply src/test/java/io/github/linuxforhealth/hl7/segments/Hl7EncounterFHIRConversionTest.java:78]: added!

File: src/test/java/io/github/linuxforhealth/hl7/segments/Hl7EncounterFHIRConversionTest.java (hunk 2 above)
Comment (jgrant22) [reply src/test/java/io/github/linuxforhealth/hl7/segments/Hl7EncounterFHIRConversionTest.java:120]: done!

**Pull request merged**